### ISSUE TITLE
Galacticraft planets compatibility

### ISSUE THREAD

**Issue by axelpx** [LemADEC/WarpDrive#208]
Is it possible to allow ships to warp into Galacticraft planets dimensions? 
Can you explain how it can be done or just show the part of the config where it can be set?

Thanks in advance!

**Comment by LemADEC** [comment]
Look for planets in the configuration file to add other dimensions, including GC ones

**Comment by axelpx** [comment]
I suppose this part is a config for plantes. But I dont quiet understand it.

```
##########################################################################################################
# planets
#--------------------------------------------------------------------------------------------------------#
# Planets are other dimensions connected through the Space dimension. Default is overworld with 100k radius.
# Each planet orbit is square shaped and defined as a list of 7 integers (all measured in blocks).
##########################################################################################################

planets {
    # dimensionId, dimensionCenterX, dimensionCenterZ, radiusX, radiusZ, spaceCenterX, spaceCenterZ
    I:overworld <
        0
        0
        0
        100000
        100000
        0
        0
     >
}
```

What does planet radius represent? Given dimension borders? So I add something like this:

```
planets {
    # dimensionId, dimensionCenterX, dimensionCenterZ, radiusX, radiusZ, spaceCenterX, spaceCenterZ
    I:overworld <
        0
        0
        0
        100000
        100000
        0
        0
     >
    I:DIM-13 <
        0
        0
        0
        10000
        10000
        0
        0
     >
}
```

It would allow me to warp to DIM-13 with 10k world radius, right?

**Comment by LemADEC** [comment]
You've created another overworld inside the existing one.
The comments gives you the description, for example the first integer value is the dimensionId. You left it to 0, hence it's the overworld.
Also, both portals are overlapping.

**Comment by axelpx** [comment]
So:
dimensionId = dimension ID (0,1,-1 or something like -13)
dimensionCenterX, dimensionCenterZ - center of the dimension INSIDE the dimesion itself?
radiusX, radiusZ - world borders of the dimension?
spaceCenterX, spaceCenterZ - coordinates of the dimension in Space Dimension?

Thanks a lot for your help and responses, but its not quite clear for me. Maybe those comments in config are not enough for people like me and you should explain it a bit more in-depth :)

**Comment by LemADEC** [comment]
you've got it right.

**Comment by axelpx** [comment]
Oh thanks :) I'd recommend adding this as comments to your config for other dumb ppl like me :D

**Comment by LemADEC** [comment]
Thanks for the heads up.
It's already fixed in 1.10 dev :).
